src/solver/optimisation/adequacy_patch_csr/construct_problem_variables.cpp

```diff
@@ -65,14 +67,14 @@ void constructVariableENS(PROBLEME_HEBDO* ProblemeHebdo, HourlyCSRProblem& hourl
     }
 }
 
-void constructVariableSpilledEnergy(PROBLEME_HEBDO* ProblemeHebdo,
+void constructVariableSpilledEnergy(const PROBLEME_HEBDO* ProblemeHebdo,
+                                    PROBLEME_ANTARES_A_RESOUDRE& ProblemeAResoudre,
                                     HourlyCSRProblem& hourlyCsrProblem)
 {
     int hour = hourlyCsrProblem.hourInWeekTriggeredCsr;
-    PROBLEME_ANTARES_A_RESOUDRE* ProblemeAResoudre = ProblemeHebdo->ProblemeAResoudre;
-    CORRESPONDANCES_DES_VARIABLES* CorrespondanceVarNativesVarOptim
-        = ProblemeHebdo->CorrespondanceVarNativesVarOptim[hour];
-    int& NumberOfVariables = ProblemeAResoudre->NombreDeVariables;
+    CORRESPONDANCES_DES_VARIABLES* CorrespondanceVarNativesVarOptim;
```

Comment:
It's considered better practice to have declaration and assignment on the same line

Reply:
done